OJP/OJP_PlaceSupport.xsd

```diff
@@ -430,6 +430,11 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:element name="GeoPosition" type="siri:LocationStructure"/>
+			<xs:element ref="SituationFullRef" minOccurs="0" maxOccurs="unbounded">
```

Comment:
Probably add a wrapper since it is unbounded ...

Reply:
wrappers done